exporter/datadogexporter/metrics_exporter.go

```diff
@@ -159,6 +168,45 @@ func (exp *metricsExporter) PushMetricsDataScrubbed(ctx context.Context, md pmet
 	return exp.scrubber.Scrub(exp.PushMetricsData(ctx, md))
 }
 
+// otelNamespacePrefix specifies the namespace used for OpenTelemetry host metrics.
+const otelNamespacePrefix = "otel."
+
+// prepareSystemMetrics prepends system hosts metrics with the otel.* prefix to identify
+// them as part of the Datadog OpenTelemetry Integration.
+func (exp *metricsExporter) prepareSystemMetrics(ms []datadog.Metric) {
+	sptr := func(s string) *string { return &s }
+	dptr := func(d int) *int { return &d }
+	for i, m := range ms {
+		name := *m.Metric
+		if !strings.HasPrefix(name, "system.") &&
+			!strings.HasPrefix(name, "process.") {
+			// not a system metric
+			continue
+		}
+		if exp.infraTransformer.Has(name) {
+			// it is a system metric, but it's a Datadog native one,
+			// so we skip it
+			switch name {
+			case "system.net.bytes_rcvd", "system.net.bytes_sent":
+				// These two metrics are reported in Datadog as Gauges, but OpenTelemetry
+				// sends them as Counts, so we need to fix this inconsistency here to avoid
+				// bugs in mixed deployments.
+				ms[i].Type = sptr("gauge")
+				fallthrough
+			case "system.load.1", "system.load.5", "system.load.15",
+				"system.cpu.idle", "system.cpu.user", "system.cpu.system",
+				"system.mem.total", "system.mem.usable", "system.swap.free",
+				"system.swap.used":
```

Comment:
nit : does it make sense to create a const for these ?

Reply:
Well, making a constant for each is tedious. Making a `[]string` variable (slices can't be constants) will imply iterating over them which is slower. I think it's fine to have a list. As long as it's double checked by tests it should be ok.